packages/shared/src/components/containers/FeedFilterStep.tsx

```diff
@@ -8,30 +8,38 @@ interface ClassName {
 }
 
 interface FeedFilterStepProps {
+  topIcon?: ReactNode;
   title: string | ReactNode;
-  description: string;
+  description?: string;
   children: ReactNode;
   className?: ClassName;
 }
 
-const FeedFilterHeading = classed('h3', 'text-center typo-title2 font-bold');
+const FeedFilterHeading = classed(
+  'h3',
+  'text-center typo-title2 font-bold px-4',
+);
 
```

Comment:
More of a generic note, I noticed this is now in `container` perhaps we can consider making `components/feedFilters` as it will receive some more components.
This specific thing can then go into a `common` file?

But ofcourse non-breaking, leaving it up to you.

Reply:
Will have this in a separate PR, will contain the whole onboarding process and be containing them in a folder named `onboarding`